test/y2partitioner/actions/resize_blk_device_test.rb

```diff
@@ -275,6 +275,17 @@ def create_partition(disk_name)
       end
     end
 
+    context "when executed on an 'empty' partition" do
+      let(:scenario) { "mixed_disks.yml" }
+
+      let(:device_name) { "/dev/sdb7" }
+
+      it "shows a warning popup" do
+        expect(Yast2::Popup).to receive(:show)
```

Comment:
I would make sure that the correct popup is used:

```
        expect(Yast2::Popup).to receive(:show).with(/No file system nor any storage system was detected/, anything)
```

Reply:
Sure, I will also make that 20 lines further down.